### DISCUSSION TITLE
Two devices with one license?

### DISCUSSION THREAD

**Discussion by matze9090** [aclap-dev/video-downloadhelper#1091] [Help and Q&A]
### Your question

I have a notebook and pc.
Can i use one license for both devices?

### Addon & CoApp version and details

version: 9.0.2.12
target: microsoft
channel: stable
lang: de
coapp: {"found":true,"new_version":false,"path":"C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe","version":"2.0.19"}
license: {"accepted":true,"email":"<removed>","key":"7d7022d1d37c426f"}
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/127.0.0.0 Safari/537.36 Edg/127.0.0.0
{
    "valid_license_message_has_been_displayed": true,
    "used_sidebar_button": true,
    "used_popup_button": true,
    "download_directory": "C:\\Users\\Matze\\Desktop",
    "last_download_directory": {
        "__serializer_tag": "option_some",
        "__serializer_value": {
            "__serializer_tag": "primitive",
            "__serializer_value": "C:\\Users\\Matze\\Desktop"
        }
    },
    "show_success_notification": false,
    "view_options": {
        "all_tabs": false,
        "hide_downloaded": false,
        "low_quality": false,
        "show_button_clean": false,
        "show_button_clean_all": true,
        "show_button_convert_local": false,
        "sort_by_status": true,
        "sort_reverse": false
    },
    "use_wide_ui": true,
    "use_sidebar": false,
    "open_count_store": 3,
    "successfull_dl": 4
}

**Comment by GrampaWildWilly** [comment]
I do not work for VDH.  I'm just another user.

Yes.  The license is for a version of VDH on a particular browser.  So you could use your license for VDH for Edge on every computer you own now & every computer you will ever own on the Edge browser.  That would be for Edge on Windows, Mac, & Linux.  Actually, I don't know if there is an Edge for Mac or Linux but if there is, the same license would work.  Well, there are no licenses on Linux, but, whatever.  I do know Firefox exists on all 3 operating systems & a license for VDH for Firefox would work on Windows & Mac, but there are no licenses on Linux so that's moot.  The only situation in which you would need an additional license is if you were to start using a second browser.  You're using Edge.  If you were to start using Firefox, you would need a second license for that.  Well, maybe need is too strong a word.  VDH works the same with & without a license.  Absence of a license simply means some of your downloads will have watermarks.  And there are other minor restrictions under certain circumstances.  But getting a license does not unlock new functions nor does it make VDH work on more web sites.  The license is primarily about skipping the insertion of the watermark.

Whatever license you have is for life.  Your life.  There is no renewal or maintenance fee.  You buy the license one time & that's your last expenditure.  But don't share it with your friends & family.  Make them buy their own licenses.
